.github/workflows/dependency-tree-update.yml

```diff
@@ -55,6 +55,8 @@ jobs:
           elif [ -f yarn.lock ]; then
             rm yarn.lock;
             yarn;
+          elif [ -f package.json ]; then
+            npm install;
```

Comment:
I guess since we're covering our bases, let's make it explode if it finds neither.
Suggest adding maybe

```
else
	echo "No package files were found. Maybe a path error?"
	exit 1;
fi;